osu.Game/Screens/OnlinePlay/Lounge/Components/RoomsContainer.cs

```diff
@@ -124,7 +124,12 @@ private void removeRooms(IEnumerable<Room> rooms)
         private void updateSorting()
         {
             foreach (var room in roomFlow)
-                roomFlow.SetLayoutPosition(room, -(room.Room.RoomID.Value ?? 0));
+            {
+                roomFlow.SetLayoutPosition(room, room.Room.Category.Value == RoomCategory.Spotlight
+                    // Always show spotlight playlists at the top of the listing.
+                    ? float.MinValue
```

Comment:
The ordering among spotlight items is a bit undefined here if there are ever more than one of them at a time, but for now that's never the case, so I'm purposefully looking away.